opentelemetry-sdk/src/opentelemetry/sdk/trace/export/__init__.py

```diff
@@ -151,6 +133,8 @@ class BatchSpanProcessor(SpanProcessor):
     - :envvar:`OTEL_BSP_MAX_QUEUE_SIZE`
     - :envvar:`OTEL_BSP_MAX_EXPORT_BATCH_SIZE`
     - :envvar:`OTEL_BSP_EXPORT_TIMEOUT`
+
+    All the logic for emitting spans, shutting down etc. resides in the BatchProcessor class.
```

Comment:
```suggestion
    All the logic for emitting spans, shutting down etc. resides in the `BatchProcessor` class.
```

(this should make sphinx docs link)

Reply:
Done thanks